.github/workflows/ci.yaml

```diff
@@ -0,0 +1,148 @@
+name: Meshery Operator Template
```

Comment:
I think the name should be Meshery Operator

Reply:
Done. Thanks!